### DISCUSSION TITLE
When you are going to support okhttp latest version 4.X ?

### DISCUSSION THREAD

**Discussion by lokeshdeshmukh** [square/retrofit#3593] [Q&A]
Is there any plan to support  OKHTTP 4.X version ?

**Comment by JakeWharton** [comment]
OkHttp 4 is binary compatible with 3 so it is already supported. Add an explicit dependency on it and your build tool will correctly only resolve a single copy. We retain a dependency on 3 for the benefit of Java users who don't want a Kotlin dependency.
